tasks/skylab/FastqProcessing.wdl

```diff
@@ -100,12 +100,13 @@ task FastqProcessing {
     fi
 
     fastqprocess \
-        --bam-size 1.0 \
+        --bam-size 30.0 \
```

Comment:
Might make more sense to call this `output-size` instead of bam-size (if it's not too much work to change)

Reply:
We need to make a new PR to sctools. I will make a note to do this change later.